src/sentry/replays/usecases/ingest/dom_index.py

```diff
@@ -396,6 +396,15 @@ def _handle_breadcrumb(
 ) -> ReplayActionsEventPayloadClick | None:
 
     click = None
+    project = Project.objects.get(id=project_id)
```

Comment:
This will blow up the database.  You're running the query for every segment ingested (2k RPS) plus every breadcrumb in the recording data.  This is going to query the database anywhere from 20k times per second to 200k times per second.  Or more realistically, we won't query at that rate and will begin to backlog as the database becomes the primary choke point.  This approach won't work.